magento2php.properties

```diff
@@ -0,0 +1,15 @@
+category=External Analysers
+description=Adds rules for Magento2 PHP development
+homepageUrl=https://github.com/rostilos/sonarqube-magento2-rules
+archivedVersions=
+publicVersions=1.3.0
+
+defaults.mavenGroupId=org.rostilos.sonarrules
+defaults.mavenArtifactId=sonar-magento2-rules
+
+1.3.0.description=PHP Custom Rules for Magento 2 CMS
+1.3.0.sqs=[10.6,24.12]
```

Comment:
This `sqs` argument declares the plugin version's compatibility with SonarQube server, the versioning of which now looks like 2025.1 ... So this should be something like 
`1.3.0.sqs=[2025.1,LATEST]`
Note that I suggest starting this at the current LTA rather than the EOL 10.6 version since this is a brand new plugin, so you can't already have users on that older version